app/views/home/_error_position_unavailable.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class='modal fade' id='error-position_unavailable' tabindex='-1' role='dialog' aria-labelledby='mySmallModalLabel'>
+  <div class='modal-dialog modal-lg'>
+    <div class='modal-content'>
+      <div class='modal-header'>
+        <%= button_tag '×', type: 'button', class: 'close', data: { dismiss: 'modal' }, aria: { hidden: 'true' } %>
+        <h4 class='text-center'>You are using Safari.</h4>
+      </div>
+
+      <div class='modal-body'>
+      <div class='row'>
+          <div class='col-md-12'>
+            <h5 class='text-center'>
+              <b>Safari Browser:</b> Please Turn on WiFi and Location Services. You can always use Chrome or Firefox to run the test.
```

Comment:
See Matt's response at https://app.slack.com/team/UKLBXKQH5. Please adjust the grammar as needed.

Reply:
`Safari requires that WiFi be turned on and your Location Services enabled. Alternatively, you can run this test using a different web browser including, Chrome and Firefox.`